receiver/collectdreceiver/testdata/config.yaml

```diff
@@ -0,0 +1,20 @@
+receivers:
+  collectd:
+  collectd/one:
```

Comment:
nit: add comments here to the configuration options so users can refer/change it to get started.